frontend/src/components/jobs/result/utils/sections.jsx

```diff
@@ -111,6 +112,10 @@ export function JobActionsBar({ job }) {
 }
 
 export function JobInfoCard({ job }) {
+  const process_time2 = new Date(job.process_time * 1000)
+    .toISOString()
+    .substring(14, 19);
```

Comment:
can you show me with which criteria do you have chosen these numbers? Code like this seems non-maintanable because you created your own parser for the date. I am sure there is another way to do this by selecting the minutes and the seconds by leveraging the libraries

Reply:
yea sure, here's how i have chosen these numbers.

<img width="550" alt="Screenshot 2022-12-24 at 2 09 18 PM" src="https://user-images.githubusercontent.com/90976669/209428084-fbc36c0d-8f8b-46fb-803c-cfd6a97c6091.png">


I can use the `moment.js` library to achieve the same, but it seems to me like a bit of overkill here, though if it's something that's more maintainable i'll use that, lemme know.